contrib/parseq-zk-client/src/main/java/com/linkedin/parseq/zk/client/ZKClient.java

```diff
@@ -302,8 +304,34 @@ protected Promise run(Context context) throws Throwable {
    * @param executor {@code Executor} that will be used to run the operations.
    * @return task to execute multiple operations.
    */
-  public Task<List<OpResult>> multi(Iterable<Op> ops, Executor executor) {
-    return Task.blocking(() -> _zkClient.multi(ops), executor);
+  public Task<List<OpResult>> multi(List<Op> ops, Executor executor) {
```

Comment:
After thinking a bit more, I decided to revert this change because it would erase the partial results of finished operations.